grails-app/controllers/au/org/emii/aatams/SensorController.groovy

```diff
@@ -32,8 +32,6 @@ class SensorController extends ReportController {
         def tag = tagFactoryService.lookupOrCreate(params.tag)
         assert(tag)
 
-        params.tag = tag
```

Comment:
For reference, https://github.com/aodn/aatams/blob/master/test/unit/au/org/emii/aatams/SensorControllerTests.groovy#L124

Do we understand why the code stopped working?